client/landing/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -1,42 +1,45 @@
 // Menu links
-.sidebar__menu-link {
-	padding-top: 14px;
-	padding-bottom: 14px;
-	font-weight: 600;
+.sidebar__jetpack-cloud {
 
-	.selected &,
-	.selected &:hover {
-		background-color: rgba( var( --color-sidebar-menu-selected-background-rgb ), 0.12 );
-	}
-}
+	.sidebar__menu-link {
+		// padding-top: 14px;
```

Comment:
Perhaps we should just remove those instead of commenting them out? Or perhaps we could add a clarifying comment, explaining why they're currently commented out?

Reply:
yeah I think I was testing something. I will add it back for now. So that it is more consistant with the previous jetpack cloud design.